ai-service/src/ai_service/handlers/ingest.py

```
@@ -0,0 +1,71 @@
+
+class IngestRequest(BaseModel):
+    repo_url: str
+
+
+def ingest_github_project(repo_url: str) -> None:
+    # Generate collection_name from repo_url
+    collection_name = db.generate_collection_name(repo_url)
+    project_dir = project_ingestor.clone_github_repo(repo_url)
+    try:
+        code_files = project_ingestor.scan_code_files(project_dir)
+        logger.info(f"Found {len(code_files)} code files to process.")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)